transport/responseerrors/errormapper.go

```diff
@@ -0,0 +1,77 @@
+package responseerrors
```

Comment:
To avoid import conflicts with the `errors` package, the package is still named `responseerrors`.

Reply:
I am also considering shorter name `resperrors` to avoid double e - `responsEEerrors`. What do you think @bolshakoff ?

Reply:
Since `transport` and `service` are different packages, there shouldn't be a conflict, and I'd have the same name for them.
And during import yes, they will have aliases like `apierrors` and `resperrors`.